### PR TITLE
Fix broken references to xlsx files + update instructions

### LINKED ISSUE #238: Is the link to Master Parts List broken?
I gt a 404 message.  Thanks!  Would love to try this with my 9th-grade students.

### PR DESCRIPTION
**Pull request by Achllle**
Close #238

### CODE REVIEW

**Comment by Achllle** [comment]
@ericjunkins ready for review

File: README.md [hunk 1]
```diff
@@ -176,11 +176,6 @@ Above is an example roadmap of how you can build the rover and which parts of th
 
  * __Stage 7:__ Add your own upgrades! We chose Raspberry Pi as the brain of the project so that it should be easy to add, change, and upgrade to build exciting things on top of this already cool robot.  Some upgrade ideas to get you brainstorming: sonar for collision detection, IMU for orientation / closed-loop driving / obstacle mapping, camera for object identification and tracking, sensor packages (temperature, pressure, humidity), solar panels, or even a robotic arm!
 
-## Building the Master Parts List
```
Comment (apollokit) [inline README.md:179]: Can you write the justification here for removing the "building the master parts list" section? Just so we keep that for posterity.

I think it was that now that we moved over to read me format, the `init.sh` script no longer works, and it's not worth updating. Correct?

**Review by apollokit — COMMENTED** [review]
Links look good. Can you add that little bit of justification I requested? Then i'd approve.

File: README.md (hunk 1 above)
Comment (apollokit) [reply README.md:179]: To be clear, I mean just write it on this pull request, so it's recorded officially.

Comment (Achllle) [reply README.md:179]: That's right. I also couldn't get the script to work last times I made changes, so had to add the parts manually.

**Comment by apollokit** [comment]
This seems basically done, right?

**Comment by Achllle** [comment]
Yup - just need the green check mark from either you or @ejunkins

**Review by apollokit — APPROVED** [review]

**Pull request merged**